sdk/identity/identity/package.json

```diff
@@ -97,7 +98,9 @@
     "qs": "^6.7.0",
     "stoppable": "^1.1.0",
     "tslib": "^2.0.0",
-    "uuid": "^8.3.0"
+    "uuid": "^8.3.0",
+    "command-exists": "~1.2.9",
+    "@types/command-exists": "~1.2.0"
```

Comment:
@bterlson What do you think of adding this dependency? Seems clean enough: https://github.com/mathisonian/command-exists (547 downloads!).

Reply:
Do we need this in the dependencies? If it's not part of the API surface it should be fine to put it in devDependencies.

Reply:
I'll remove it! Thanks for the feedback.